servicetalk-test-resources/build.gradle

```diff
@@ -17,15 +17,16 @@
 apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
 
 dependencies {
+  api platform(project(":servicetalk-dependencies"))
   testImplementation enforcedPlatform("org.junit:junit-bom:$junit5Version")
 
   implementation project(":servicetalk-annotations")
   implementation project(":servicetalk-utils-internal")
 
-  api "org.hamcrest:hamcrest:$hamcrestVersion"
+  api "org.hamcrest:hamcrest"
 
-  implementation "org.apache.logging.log4j:log4j-core:$log4jVersion"
-  implementation "org.apache.logging.log4j:log4j-slf4j-impl:$log4jVersion"
+  implementation "org.apache.logging.log4j:log4j-core"
```

Comment:
IIUC, this line can be removed bcz `log4j-slf4j-impl` will bring it transitively

Reply:
`log4j-slf4j-impl` is runtime only so the`core` is still needed at compile time.

Reply:
Can you clarify why? There are no java code in this module that depends on `log4j-core`. We only add `log4j2.xml` file that is evaluated at runtime.

Reply:
I believed it was needed for logging API but can try removing it.

Reply:
Agreed, it was not there before #2210 and everything worked fine